WordPress/Sniffs/DB/DirectDatabaseQuerySniff.php

```diff
@@ -163,22 +159,15 @@ public function process_token( $stackPtr ) {
 			}
 		}
 
-		// Flag instance if not whitelisted.
-		if ( ! $whitelisted_db_call ) {
-			$this->phpcsFile->addWarning( 'Usage of a direct database call is discouraged.', $stackPtr, 'DirectQuery' );
-		}
+		$this->phpcsFile->addWarning( 'Usage of a direct database call is discouraged.', $stackPtr, 'DirectQuery' );
```

Comment:
```suggestion
		$this->phpcsFile->addWarning( 'Use of a direct database call is discouraged.', $stackPtr, 'DirectQuery' );
```

Reply:
Out of scope of this PR, but I've updated it anyway.